templates/sectionsummary.html

```diff
@@ -30,7 +30,8 @@ <h1>{{ content.summary.title }}</h1>
                 {% set add_link_text = summary.add_link_text %}
                 {% set empty_list_text = summary.empty_list_text %}
                 {% set list_title = summary.title %}
-                <div {{ 'class="ons-u-mb-l"' if not loop.last }} data-qa="{{ summary.list_name }}-list-summary">
```

Comment:
Why was this change added? Seems unrelated to your changes?

Reply:
Oh I think was an accidental reformat. Reverted now 👍